evaluation/fix_spans_test.py

```diff
@@ -0,0 +1,38 @@
+# Lint as: python3
+"""Tests for fix_spans."""
+
+import unittest
+
+import fix_spans
+
+
+class FixSpansTest(unittest.TestCase):
+
+    def test_leading_spaces(self):
+        self.assertEqual(fix_spans._fix_spans(
+            [2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 13, 14],
+            'To infinity and    beyond.'),
+                         [3, 4, 5, 6, 7, 8, 9, 10, 12, 13, 14])
+
+    def test_ignore_space_in_span(self):
+        self.assertEqual(fix_spans._fix_spans(
+            [3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14],
+            'To infinity and   beyond.'),
+                         [3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14])
+
+    def test_trailing_spaces(self):
+        self.assertEqual(fix_spans._fix_spans(
+            [3, 4, 5, 6, 7, 8, 9, 10, 12, 13, 14, 15, 16, 17, 19, 20, 21, 22,
```

Comment:
Here the third span [19, 20, 21, 22, 23, 24] "eyond." missed the b and does not add it back.

Reply:
While that was my mistake in choosing the offset, I am not currently moving boundaries within words, only dealing with white space. We could do that but that is a different discussion I think.

Reply:
It's clear, thanks!